### PR TITLE
Rewards Polish

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

The last of the known-to-me rewards issues. Still waiting on QA pass though. 
Should be easily reviewable by commit.

Can't repro lineup infinite load anymore 🤷 

### Dragons

Touched popover?

### How Has This Been Tested?

Locally

### CODE REVIEW

File: src/containers/App.js [hunk 1]
```diff
@@ -622,24 +620,6 @@ class App extends Component {
                   />
                 )}
               />
-              <Route
-                exact
-                path={EXPLORE_TOP_ALBUMS_PAGE}
```
Comment (raymondjacobson) [inline src/containers/App.js:627]: no more top albums?

File: src/containers/audio-rewards-page/components/modals/TransferAudioMobileDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import Drawer from 'components/drawer/Drawer'
+import React from 'react'
+import { useModalState } from 'store/application/ui/modals/hooks'
+import { ReactComponent as IconGold } from 'assets/img/IconGoldBadge.svg'
+import styles from './TransferAudioMobileDrawer.module.css'
+
+const messages = {
+  title: 'Transfer $AUDIO',
+  subtitle: 'To transfer AUDIO please visit audio.co from a desktop browser'
```
Comment (raymondjacobson) [inline src/containers/audio-rewards-page/components/modals/TransferAudioMobileDrawer.tsx:9]: audius.co

File: src/containers/explore-page/components/desktop/Section.tsx [hunk 3]
```diff
@@ -13,8 +13,7 @@ export enum Layout {
   NORMAL = 'normal',
   // Two-columns with an alternating pattern of wide, medium, and narrow
   TWO_COLUMN_DYNAMIC = 'two-column-dynamic',
-  // Two-columns with an alternating pattern of wide, medium, and narrow
-  // that alternates with a single column every 9 items,
+  // Two-columns with rows of 100%x1, 50%x2, 33%x3, 67% 33% in that order
```
Comment (raymondjacobson) [inline src/containers/explore-page/components/desktop/Section.tsx:16]: nice

File: src/containers/explore-page/store/types.ts [hunk 4]
```diff
@@ -23,7 +23,7 @@ export default interface ExplorePageState {
 export enum ExploreCollectionsVariant {
   LET_THEM_DJ = 'Let Them DJ',
   TOP_ALBUMS = 'Top Albums',
-  TOP_PLAYLISTS = 'Top Playlists',
   MOOD = 'Mood',
-  DIRECT_LINK = 'Direct Link'
+  DIRECT_LINK = 'Direct Link',
+  TOP_PLAYLISTS = 'Top Playlists'
```
Comment (raymondjacobson) [inline src/containers/explore-page/store/types.ts:28]: worth re-typing as TRENDING_PLAYLISTS?

File: src/containers/audio-rewards-page/components/modals/RewardsModals.tsx [hunk 5]
```diff
@@ -9,6 +10,7 @@ const RewardsModals = () => {
       <TrendingRewardsModal />
       <VerifiedUpload />
       <TopAPIModal />
+      <TransferAudioMobileDrawer />
```
Comment (raymondjacobson) [inline src/containers/audio-rewards-page/components/modals/RewardsModals.tsx:13]: could condition this on isMobile

File: src/components/general/Popup.js [hunk 6]
```diff
@@ -132,7 +132,7 @@ const Popup = ({
             break
           case 'bottomRight':
             top = rect.y
-            left = rect.x
+            left = rect.x + triggerRect.width
```
Comment (raymondjacobson) [inline src/components/general/Popup.js:135]: i *think* this is probably good for the other use case, but to confirm with @sddioulde

**Review by raymondjacobson — APPROVED** [review]
good looks! ill help do some UI QA after we merge/whatnot this guy

File: src/containers/App.js (hunk 1 above)
Comment (piazzatron) [reply src/containers/App.js:627]: whoa total accident nice catch

File: src/containers/audio-rewards-page/components/modals/TransferAudioMobileDrawer.tsx (hunk 2 above)
Comment (raymondjacobson) [reply src/containers/audio-rewards-page/components/modals/TransferAudioMobileDrawer.tsx:9]: also maybe copy shold be to transfer $AUDIO?

Comment (piazzatron) [reply src/containers/audio-rewards-page/components/modals/TransferAudioMobileDrawer.tsx:9]: heh

File: src/containers/explore-page/store/types.ts (hunk 4 above)
Comment (piazzatron) [reply src/containers/explore-page/store/types.ts:28]: yeah i'll retype, was being lazy

File: src/components/general/Popup.js (hunk 6 above)
Comment (piazzatron) [reply src/components/general/Popup.js:135]: yeah sg. fyi this was feedback I got from Julian about the rewards use case - not sure that it applies to profile but I suspect it does.

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/piazz-rewards-fixes

File: src/components/general/Popup.js (hunk 6 above)
Comment (sddioulde) [reply src/components/general/Popup.js:135]: Hmm. Depends on what bottom right really means for us. Wouldn't be the end of the world for sort use case, but let's confirm

Comment (piazzatron) [reply src/components/general/Popup.js:135]: resolved on slack

**Pull request merged**